htdocs/admin/facture.php

```diff
@@ -364,7 +364,7 @@
 							if ($conf->global->FACTURE_ADDON == $file || $conf->global->FACTURE_ADDON.'.php' == $file) {
 								print img_picto($langs->trans("Activated"), 'switch_on');
 							} else {
-								print '<a class="reposition" href="'.$_SERVER["PHP_SELF"].'?action=setmod&token='.newToken().'&value='.preg_replace('/\.php$/', '', $file).'&scan_dir='.$module->scandir.'&label='.urlencode($module->name).'" alt="'.$langs->trans("Default").'">'.img_picto($langs->trans("Disabled"), 'switch_off').'</a>';
```

Comment:
Why removing scan_dir= param ?
Is it done on purpose ?
It is used for ODT generator when a var to store paths for ODT files is required.

Reply:
@eldy The occurrences I removed are in the context of numbering modules, where this property doesn't exist, and not ODT/PDF modules. I guess it was there because of a bad copy/paste.